Content.Client/Implants/UI/ChameleonControllerMenu.xaml.cs

```diff
@@ -0,0 +1,77 @@
+using System.Numerics;
+using Content.Client.Stylesheets;
+using Content.Shared.Roles;
+using Robust.Client.AutoGenerated;
+using Robust.Client.GameObjects;
+using Robust.Client.UserInterface.Controls;
+using Robust.Client.UserInterface.CustomControls;
+using Robust.Client.UserInterface.XAML;
+using Robust.Shared.Prototypes;
+
+namespace Content.Client.Implants.UI;
+
+[GenerateTypedNameReferences]
+public sealed partial class ChameleonControllerMenu : DefaultWindow
```

Comment:
You should probably be using `FancyWindow` instead of `DefaultWindow`.

DefaultWindow is from RT and has slightly different styling and doesn't have some of the changes made for SS14 specifically (e.g. help button).